src/Azure.Functions.Cli/Helpers/WorkerRuntimeLanguageHelper.cs

```diff
@@ -208,5 +207,10 @@ public static bool IsDotnetIsolated(WorkerRuntime worker)
         {
             return worker ==  WorkerRuntime.dotnetIsolated;
         }
+
+        internal static string GetAvailableOptions()
+        {
+            return string.Join(", ", normalizeMap.Keys.Where(k => !string.Equals(k, "None", StringComparison.OrdinalIgnoreCase)));
```

Comment:
nit: "None" should be a const

Reply:
Hi @aishwaryabh , the code mentioned above has been removed, as it is no longer required.